Formula/pdns.rb

```diff
@@ -26,54 +27,79 @@ class Pdns < Formula
   deprecated_option "with-pgsql" => "with-postgresql"
 
   depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
   depends_on "boost"
   depends_on "lua"
   depends_on "openssl"
   depends_on "sqlite"
   depends_on :postgresql => :optional
 
+  resource "pdns-brew-default-config" do
+    url "https://github.com/mprzybylski/pdns-brew-default-config/archive/0.2.tar.gz"
```

Comment:
If you want to go that route, have at it.  I don't have any more time to work on it, so I'm just going to yank it out.